modules/mono/editor/GodotTools/GodotTools/Build/BuildOutputView.cs

```diff
@@ -1,425 +1,150 @@
 using Godot;
-using System;
-using System.Diagnostics.CodeAnalysis;
-using GodotTools.Internals;
-using File = GodotTools.Utils.File;
-using Path = System.IO.Path;
+using static GodotTools.Internals.Globals;
+
+#nullable enable
 
 namespace GodotTools.Build
 {
-    public partial class BuildOutputView : VBoxContainer, ISerializationListener
+    public partial class BuildOutputView : HBoxContainer
     {
-        [Serializable]
-        private partial class BuildIssue : RefCounted // TODO Remove RefCounted once we have proper serialization
-        {
-            public bool Warning { get; set; }
-            public string File { get; set; }
-            public int Line { get; set; }
-            public int Column { get; set; }
-            public string Code { get; set; }
-            public string Message { get; set; }
-            public string ProjectFile { get; set; }
-        }
-
-        [Signal]
-        public delegate void BuildStateChangedEventHandler();
-
-        public bool HasBuildExited { get; private set; } = false;
+#nullable disable
+        private RichTextLabel _log;
 
-        public BuildResult? BuildResult { get; private set; } = null;
+        private Button _clearButton;
+        private Button _copyButton;
+#nullable enable
 
-        public int ErrorCount { get; private set; } = 0;
-
-        public int WarningCount { get; private set; } = 0;
-
-        public bool ErrorsVisible { get; set; } = true;
-        public bool WarningsVisible { get; set; } = true;
-
-        public Texture2D BuildStateIcon
+        public void Append(string text)
         {
-            get
-            {
-                if (!HasBuildExited)
-                    return GetThemeIcon("Stop", "EditorIcons");
-
-                if (BuildResult == Build.BuildResult.Error)
-                    return GetThemeIcon("Error", "EditorIcons");
-
-                if (WarningCount > 1)
-                    return GetThemeIcon("Warning", "EditorIcons");
-
-                return null;
-            }
+            _log.Text += text;
```

Comment:
I think it should be better to use `RichTextLabel::add_text`, unless this is somehow overloaded?